src/components/Contentful/DynamicHero.vue

```diff
@@ -235,8 +235,9 @@ export default {
 			};
 		},
 		heroBackground() {
-			// TODO: use the contentType param to target this content instead of a string fragment in the key
-			return this.content?.contents?.find(({ key }) => key.indexOf('background') > -1);
+			return this.content?.contents?.find(({ contentType }) => {
```

Comment:
There's currently an issue in dev wherein we're not checking for `key` as it's not initially present so it's breaking the page.

I don't think the contentType can be undefined but am checking anyway...